doc/cookbook/source/examples/classifier/kmeans.rst

```diff
@@ -0,0 +1,47 @@
+==================
+:math:`k`-means clustering
+==================
+:math:`k`-means clustering aims to partition :math:`n` observations into :math:`k` (:math:`\leq n`) clusters in which each observation belongs to the cluster with the nearest mean, serving as a prototype of the cluster.
+
+The :math:`n` observations are represented by :math:`n` :math:`d`-dimensional real vecoters, :math:`\mathbf{x} = (x_1, x_2, ..., x_n)`.
+
+
+
+In :math:`k`-means clustering, the :math:`n` observations will partitioned into :math:`k` (:math:`\leq n`) sets :math:`\mathbf{S} = {S_1, S_2, ..., S_k}`, with minimal within-cluster sum of squares (WCSS) (sum of distance functions of each point in the cluster to the :math:`k^{th}` center). 
+In other words, its objective is to find:
+
+.. math::
+   k = \underset{\mathbf{S}}{argmin} \sum_{i=1}^{k}\sum_{\mathbf{x}\in S_k}\left \|\boldsymbol{x} - \boldsymbol{\mu}_i  \right \|^{2}
```

Comment:
Do you mean `\underset{\mathbf{S}}{\argmin}` ?

The online LaTeX editor I'm using return blank for `\argmin` but `\arg\min` works...I guess it is somehow outdated?